bin/reth/src/lib.rs

```diff
@@ -49,33 +51,43 @@ struct NetworkOpts {
 #[derive(Debug, Args)]
 #[command(next_help_heading = "Rpc")]
 struct RpcServerOpts {
-    #[arg(long, default_value = "true")]
+    /// Enable the HTTP-RPC server
+    #[arg(long, default_value = "false")]
     http: bool,
 
+    /// Http server address to listen on
     #[arg(long = "http.addr")]
-    http_addr: Option<String>,
+    http_addr: Option<IpAddr>,
 
+    /// Http server port to listen on
     #[arg(long = "http.port")]
-    http_port: Option<u128>,
+    http_port: Option<u16>,
 
-    #[arg(long = "http.api", value_delimiter = ',')]
-    http_api: Option<Vec<RethRpcModule>>,
+    /// Rpc Modules to be configured for http server
+    #[arg(long = "http.api")]
+    http_api: Option<RpcModuleConfig>,
 
-    #[arg(long, default_value = "true")]
+    /// Enable the WS-RPC server
+    #[arg(long, default_value = "false")]
     ws: bool,
 
+    /// Ws server address to listen on
     #[arg(long = "ws.addr")]
-    ws_addr: Option<String>,
+    ws_addr: Option<IpAddr>,
 
+    /// Http server port to listen on
     #[arg(long = "ws.port")]
-    ws_port: Option<u128>,
+    ws_port: Option<u16>,
 
-    #[arg(long = "ws.api", value_delimiter = ',')]
-    ws_api: Option<Vec<RethRpcModule>>,
+    /// Rpc Modules to be configured for Ws server
+    #[arg(long = "ws.api")]
+    ws_api: Option<RpcModuleConfig>,
 
+    /// Disable the IPC-RPC  server
     #[arg(long, default_value = "true")]
```

Comment:
having a bool flag with default true isn't very useful
```suggestion
    #[arg(long)]
```